meshtastic/tests/test_node_runtime_transport.py

```
@@ -0,0 +1,1236 @@
+
+        mock_local_node.ensureSessionKey.assert_called_once()
+        call_args = mock_local_node._send_admin.call_args
+        admin_message = call_args[0][0]
+
+        assert admin_message.remove_fixed_position is True
+
+    @pytest.mark.unit
+    def test_remove_fixed_position_logs_info(
+        self,
+        position_time_runtime: _NodePositionTimeCommandRuntime,
+        _mock_local_node: MagicMock,
```

Ruff (0.15.6)

[warning] Fixture `_mock_local_node` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)